funnel/assets/js/proposal.js

```diff
@@ -1,13 +1,28 @@
+import { Utils } from './util';
+
 export const Proposal = {
   init() {
-    $('button[name="transition"][value="delete"]').click(function (e) {
-      if (
-        !window.confirm(
-          window.gettext('Do you really want to delete this proposal?')
```

Comment:
Where has this text gone?

Reply:
Proposal delete confirmation page takes care of this.